stage-ovos/02-ovos/files/mycroft.conf

```diff
@@ -1,66 +1,68 @@
-         "samples": ["hey mycroft", "hey microsoft", "hey mike roft", "hey minecraft"],
-         "rule": "fuzzy",
-         "listen": true,
-         "fallback_ww": "hey_mycroft_pocketsphinx"
-         },
-      "hey_mycroft_pocketsphinx": {
-         "module": "ovos-ww-plugin-pocketsphinx",
-         "phonemes": "HH EY . M AY K R AO F T",
-         "threshold": 1e-90,
-         "lang": "en-us",
-         "listen": true
-         }
-      },
-   "stt": {
-      "fallback_module": ""
-      }
+    "ready_settings": ["skills"],
+    "confirm_listening": true,
+    "play_wav_cmdline": "paplay %1",
+    "play_mp3_cmdline": "mpg123 %1",
+    "ipc_path": "/ramdisk/mycroft/ipc/",
+    "network_tests": {
+        "ip_url": "https://api.ipify.org",
+        "dns_primary": "1.1.1.1",
+        "dns_secondary": "8.8.8.8",
+        "web_url": "http://nmcheck.gnome.org/check_network_status.txt",
+        "web_url_secondary": "https://checkonline.home-assistant.io/online.txt",
+        "captive_portal_url": "http://nmcheck.gnome.org/check_network_status.txt",
+        "captive_portal_text": "NetworkManager is online"
+        },
+    "listener": {
+        "mute_during_output": false,
+        "instant_listen": true,
+        "VAD": {
+            "module": "ovos-vad-plugin-silero",
+            "ovos-vad-plugin-silero": {"threshold": 0.2},
+            "ovos-vad-plugin-webrtcvad": {"vad_mode": 3}
+            },
+        "retry_mic_init" : false,
+        "wake_word_upload": {
+            "disable": true}
+    },
+    "precise": {
+        "use_precise": false
+        },
+    "hotwords": {
+        "hey_mycroft": {
+            "module": "ovos-ww-plugin-openwakeword",
+            "models": ["/home/ovos/.local/share/openwakeword/hey_mycroft_v0.1.onnx"],
+            "threshold": 0.4
+            },
+        "hey_mycroft_vosk": {
+            "module": "ovos-ww-plugin-vosk",
+            "model_folder": "/home/ovos/.local/share/vosk/vosk-model-small-en-us-0.15",
+            "samples": ["hey mycroft", "hey microsoft", "hey mike roft", "hey minecraft"],
+            "rule": "fuzzy",
+            "listen": true,
+            "fallback_ww": "hey_mycroft_pocketsphinx"
+            },
+        "hey_mycroft_pocketsphinx": {
+            "module": "ovos-ww-plugin-pocketsphinx",
+            "phonemes": "HH EY . M AY K R AO F T",
+            "threshold": 1e-90,
+            "lang": "en-us",
+            "listen": true
+            }
+        },
+    "stt": {
+        "fallback_module": ""
+        },
+#    "tts": {
+#        "module": "ovos-tts-plugin-piper",
+#        "ovos-tts-plugin-piper": {
+#            "model": "/home/ovos/.local/share/piper/voices/en-gb-alan-low.onnx",
+#            "model_json": "/home/ovos/.local/share/piper/voices/en-gb-alan-low.onnx.json"
+#            }
+#        },
+    "skills": {
+        "directory": "skills",
+        "extra_directories": ["/home/ovos/.local/share/mycroft/skills"]
```

Comment:
This was needed for a while, but the bug has been fixed.  `extra_directories` is no longer needed

Reply:
Gone.